curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md

```diff
@@ -481,11 +481,11 @@ body {
 
 ```css
 :root {
-  --primary: blue;
+ primary: blue;
```

Comment:
```suggestion
  primary: blue;
```

Reply:
@StephenMuya I see you made changes to the PR, but you still need to incorporate this requested change